Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented:
@DimitriDR  / @gilbert-fernandes - this is the PR created by Weblate based on your changes. Can you please check if it looks alright for you? For any changes necessary, please update it via the Weblate UI, and then this PR will be eventually updated. I can also force Weblate to update the PR upon request.

weblate committed:
Update translation files

Updated by "Cleanup translation files" hook in Weblate.

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Update translation files

Updated by "Cleanup translation files" hook in Weblate.

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Co-authored-by: Dimitri <dimitridroeck@gmail.com>
Co-authored-by: Gilbert Fernandes <gilbert.fernandes@orange.fr>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Dimitri <dimitridroeck@gmail.com>
Signed-off-by: Gilbert Fernandes <gilbert.fernandes@orange.fr>
Signed-off-by: Hosted Weblate <hosted@weblate.org>

DimitriDR commented:
Looks alright for me.

ahus1 reviewed (approved):
Approving based on @DimitriDR's review. Thanks!
